benchmarks/README.md

```diff
@@ -130,10 +134,68 @@ Run verbose with only a single loop\
 
 The full list of options can be found by running with the -h or --help flags.
 
+# Benchmarking with Apache Bench
+
+## Installation 
+
+### For Ubuntu
+
+```
+apt-get install apache2-utils
+
+```
+
+Apache Bench is installed in Mac by default. You can test by running ```ab -h```
+
+## Benchmark 
+
+To run benchmarks execute benchmark script as follows 
+
+```
+./benchmark-ab.sh --model  vgg11 --url https://torchserve-mar-files.s3.amazonaws.com/vgg11.mar --bsize 1 --bdelay 50 --worker 4 --input ../examples/image_classifier/kitten.jpg --requests 20 --concurrency 10
```

Comment:
The script should stop the torchserve server once the benchmark is complete

Reply:
Fixed it.